libnopegl/src/pipeline.h

```diff
@@ -80,6 +81,7 @@ struct pipeline_params {
 };
 
 struct pipeline {
+    struct ngli_rc rc;
```

Comment:
Can we have `NGLI_STATIC_ASSERT(rc_is_first, offsetof(struct pipeline, rc) == 0);` for all these structures?

Reply:
Done.